tensordict/base.py

```diff
@@ -2914,11 +2914,12 @@ def map(
         self,
         fn: Callable,
         dim: int = 0,
-        num_workers: int = None,
-        chunksize: int = None,
-        num_chunks: int = None,
-        pool: mp.Pool = None,
-        seed: int = None,
+        num_workers: int | None = None,
+        chunksize: int | None = None,
+        num_chunks: int | None = None,
+        pool: mp.Pool | None = None,
+        seed: int | None = None,
+        maxtasksperchild: int | None = None,
```

Comment:
nit: but maybe max_tasks_per_child will be easier to read

Reply:
I know! But it's what mp.Pool is using so I wanted to be consistent. Readable inputs are probably less harmful though

Reply:
Thanks for the clarification!

Reply:
No worry I'll edit it!